html/changelogs/Sindroman-airbubble.yml

```diff
@@ -0,0 +1,8 @@
+author: PoZe
+
+delete-after: True
+
+changes:
+  - bugfix: "Airbubble no longer produces infinite cable restrains."
```

Comment:
Airbubble should be plural here, produces should be singular, and it should be `restraints`.

Reply:
It is an object though.